src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -516,246 +516,266 @@ private IVariableInformation[] ExpandVisualized(IVariableInformation variable)
                     IVariableInformation expr = GetExpression(item.Value, variable, visualizer.ScopedNames, item.Name);
```

Comment:
This file was not meant to be checked in here. Tests will validate that these changes are valid 😅